public/index.html

```diff
@@ -432,7 +487,23 @@ <h5 class="font-semibold text-xl md:text-2xl pb-2">Ending</h5>
       </div>
     </div>
     -->
-
+    <section class="bg-waves-bg-upsd aspect-video w-full bg-repeat bg-center bg-cover p-200">
+      <h3 class="text-5xl lg:text-6xl text-center text-white pt-20">Sponsors</h3>
+      <div class="mx-10">
+        <h2 class="text-3xl md:text-4xl lg:text-5xl text-center font-semibold pt-10 text-white pb-6">Gold</h2>
```

Comment:
let's merge these two headers into one, "Gold Sponsor", for now, and remove the horizontal line